samples/cm-resources-sample/gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 
 kotlin = "1.9.23"
-compose = "1.6.2"
+compose = "1.6.10-rc01"
```

Comment:
jetbrains.resources broken for ios on 1.6.2, 
On 1.6.10-rc02 we has:
https://github.com/JetBrains/compose-multiplatform/issues/4809
for worked sample chosen rc01